Silently fails/stalls webpack with no error

I have 2 imports that I use in multiple scss files that look like the following:

```
@import 'variables';
@import 'functions';
```

Everything works fine until I hit one file that has the exact same imports. Then, I see webpack build stall and it shows no errors. If I remove the import for this file, everything works, if I add it, it stalls. I don't understand why its happening because it's using the exact same imports. Problem still occurs if I remove everything in that file except for the imports. Problem does not occur if I remove the imports as well.

Dzięki, @piotrf 

Right now I am getting the same `Module not found: Error: Cannot resolve 'file' or 'directory' ./modules/TopBar/TopBar` error for 4 modules, no matter what :)

With https://github.com/saper/webpack-playground/tree/uncommenting@f547bb0 I get:

https://gist.github.com/e0ec0e85bc048922ca27

With the master (https://github.com/saper/webpack-playground/commit/f547bb09dc7cd70ccc6b752fad13942fa506ba64) is gives

https://gist.github.com/9255315d068b8edc36b

Not a big difference, if any....

--- a/index.js
+++ b/index.js
@@ -5,6 +5,7 @@ var sass = require('node-sass');
 var path = require('path');
 var os = require('os');
 var fs = require('fs');
+var async = require('async');
 
 // A typical sass error looks like this
 var SassError = {
@@ -15,6 +16,12 @@ var SassError = {
     status: 1
 };
 var resolveError = /Cannot resolve/;
+
+// This queue makes sure node-sass leaves one thread available for executing
+// fs tasks when running the custom importer code.
+// This can be removed as soon as node-sass implements a fix for this.
+var threadPoolSize = process.env.UV_THREADPOOL_SIZE || 4;
+var asyncSassJobQueue = async.queue(sass.render, threadPoolSize - 1);
 
 /**
  * The sass-loader makes node-sass available to webpack modules.
@@ -155,7 +162,8 @@ module.exports = function (content) {
             throw err;
         }
     }
-    sass.render(opt, function onRender(err, result) {
+
+    asyncSassJobQueue.push(opt, function onRender(err, result) {
         if (err) {
             formatSassError(err);
             callback(err);

--- a/package.json
+++ b/package.json
@@ -27,6 +27,7 @@
     "node-sass": "^3.1.0"
   },
   "dependencies": {
+    "async": "^1.4.0",
     "loader-utils": "^0.2.5"
   },
   "devDependencies": {

